synapseclient/client.py

```diff
@@ -2097,12 +2124,20 @@ def submit(self, evaluation, entity, name=None, team=None, silent=False, submitt
             elif team and 'name' in team:
                 submitterAlias = team['name']
 
+
+        if isinstance(entity, synapseclient.DockerRepository):
+            docker_digest = self._get_docker_digest(entity, dockerTag)
+        else:
+            docker_digest = None
+
         submission = {'evaluationId': evaluation_id,
                       'name': name,
                       'entityId': entity_id,
                       'versionNumber': entity_version,
+                      'dockerDigest': docker_digest,
+                      'dockerRepositoryName':docker_repository,
```

Comment:
Please see pep8 for space after colon (:) https://www.python.org/dev/peps/pep-0008/#variable-annotations